config/initializers/as_to_time.rb

```diff
@@ -14,4 +14,22 @@ def to_time(form = OBJ)
       old_to_time(form)
     end
   end
+
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  def parse_iec_number()
```

Comment:
Wow I didn't see this comment when I [commented below](https://github.com/ManageIQ/manageiq/pull/7955#discussion-diff-60081623), and I just happened to pick the EXACT SAME name.  Now I'm sure it's the right name. 😃